### DISCUSSION TITLE
All my Github pages redirect to a wrong website

### DISCUSSION THREAD

**Discussion by Vallata** [community/community#61273] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, 
I am a beginer with Github and code. I have bought a free DNS on Namecheap with a Github pack (alizeevallat.me). 
Now, all my Github pages redirect automatically to alizeevallat.me. 
I have tried many things recommended in the Github forums, but nothing works: 
- deleted all my cookies and cache 
- tried to connect to my github pages on another computer 
- checked the settings "Github Pages" of the repo -> it seems empty.
<img width="958" alt="Capture d’écran 2023-07-19 à 17 28 30" src="https://github.com/community/community/assets/121977267/324a8f45-2bec-449e-8bc1-478b620b9870">

I don't know what to do next. Can you help me please? :)

Thank you !

**Comment by queenofcorgis** [comment]
@Vallata ah that sounds so frustrating, especially since you've tried so many different solutions. Have you checked out this [troubleshooting guide](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/troubleshooting-custom-domains-and-github-pages) from our Docs it is specifically for Custom Domains :octocat: .

**Comment by Mariballardin** [comment]
Hey @Vallata 
I'm having the same problem. How did you fix it?

**Comment by Vallata** [comment]
i realized that the CNAME file I should delete was the general CNAME file existing in the general repository named « your gut hub name .io »

(And not a CNAME file directly included in the repository hosting my GitHub page). 
 
 hope it will help you!

**Reply by Mariballardin** [reply]
Thank you Vallata. Finally I found where is this CNAME. I just deleted but still redirecting to the namecheap domain omg!!!
Maybe it takes a few hours to work well? Or it was immediately for you?
In any case THANK YOU so much for explaining where is this annoying CNAME file. I had no ideia.

**Comment by Vallata** [comment]
I also deleted all my cookies and it helped
